src/fjall.rs

```diff
@@ -54,10 +54,8 @@ impl BenchmarkEngine<FjallClient> for FjallClientProvider {
 			.manual_journal_persist(false)
 			// Set the amount of data to build up in memory
 			.max_write_buffer_size(u64::MAX)
-			// Set the blob cache size to 256 MiB
-			.blob_cache(Arc::new(BlobCache::with_capacity_bytes(memory)))
-			// Set the block cache size to 256 MiB
-			.block_cache(Arc::new(BlockCache::with_capacity_bytes(memory)))
+			// Set the cache size to 256 MiB
+			.cache_size(memory)
```

Comment:
```suggestion
			.cache_size(memory)
```
Can we change the `MIN_CACHE_SIZE` on line 21 to: `512 * 1024 * 1024`

and then switch this to:

```suggestion
			// Set the cache size to 512 MiB
			.cache_size(memory)
```